### PR TITLE
MAID-2456: rename SectionState to NodeState

### PR DESCRIPTION
**Pull request by Fraser999**

### CODE REVIEW

File: src/vote.rs [hunk 1]
```diff
@@ -88,7 +87,7 @@ mod tests {
         unwrap!(rust_sodium::init_with_rng(&mut rng));
         let full_info = FullInfo::node_new(rng.gen_range(0, 255));
         let node_info = *full_info.public_info();
-        let payload = SectionState::Live(node_info);
+        let payload = "Live";
```
Comment (maqi) [inline src/vote.rs]: shall we use `NodeState::ElderLive` to demonstrate the way how network votes?
same to couple of other places

Comment (Fraser999) [reply src/vote.rs]: I don't think so - we should be able to use a simpler form for the unit tests here since that's what the API allows.

Comment (maqi) [reply src/vote.rs]: even in the tests of Block ?

Comment (Fraser999) [reply src/vote.rs]: Yes, I think so.  The votes, proofs and blocks should be usable for nodes to vote on arbitrary data - it's only when we start getting to data_chain that restrictions come in.

**Pull request merged**